examples/chip-tool/config/PersistentStorage.cpp

```diff
@@ -169,7 +169,7 @@ uint16_t PersistentStorage::GetListenPort()
     CHIP_ERROR err = CHIP_NO_ERROR;
 
     // By default chip-tool listens on an ephemeral port.
-    uint16_t chipListenPort = 0;
+    uint16_t chipListenPort = CHIP_PORT;
```

Comment:
Can you please make this a port that is not a well-known port? CHIP_PORT is only supposed to be used for multicast. Suggest using a forced port

Reply:
This port does not get used.  The next 3 ports get used...

Reply:
It does not matter which base port is used, it can very well be a random port. What matters is that once the port has been chosen and pairing successfully completed it should not keep changing at every invocation of chip-tool targeting that same controller. Changing ports at every invocation has resource impact on constrained devices because it causes creation of a new peer.